table/block.cc

```diff
@@ -162,7 +162,7 @@ bool BlockIter::ParseNextKey() {
   }
 
 // Binary search in restart array to find the first restart point
-// with a key >= target (TODO: this comment is inaccurate)
```

Comment:
What. FTR, this was introduced by https://reviews.facebook.net/D19047, but I'm not sure why it wasn't fixed.

cc @siying you know more about this code, is this correction correct?